cypress/integration/prospects/profile.js

```diff
@@ -24,4 +24,55 @@ describe('Prospects profile info', () => {
     cy.get('.error-title').contains('User not found')
     cy.get('.error-message').contains('The user you are searching for does not exist.')
   })
-})
\ No newline at end of file
+
+  it('Should have permissions when logged in as user', () => {
+    cy.login('normal@example.com', 'password');
+    cy.visit('http://localhost:3000/prospects/list');
+
+    // open each prospect profile by clicking on the prospect card
+    for (let index = 1; index < 6; ++index) {
+      // open the prospect profile
+      cy.get('div[id="prospect-' + index + '"').click();
+
+      // Open the profile menu and check if it is visible
+      cy.get('#profile-action').click();
+      cy.get('#delete-button').should('be.visible');
+
+      // Click the first and only item in the main menu,
+      // open the modal window and cancel the action
+      cy.get('ul li').should('have.length', 1);
+      cy.get('#delete-button').click();
+      cy.get('#modal-confirm').should('be.visible');
+      cy.get('#confirm-action-message')
+        .should('be.visible')
+        .contains('Are you sure you want to delete Prospect ' + index + '?');
+      cy.get('#cancel-action').click();
+      cy.get('#modal-confirm').should('not.exist');
+      cy.go('back');
+    }
+
+    // open each prospect profile by clicking on the prospect card
+    for (let index = 1; index < 6; ++index) {
```

Comment:
This `for` loop should go from 6 to 10, so when the first 5 users are deleted it will check the existing ones. Also, make two different `it` tests with suggestive titles.